recipes/fparser/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "fparser" %}
+{% set version = "0.0.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2b5051e43a3e782c30f85cacabdb597737b8ec1205a91032d7584b0e61f27a2f
+
+build:
+  entry_points:
+    - fparser2=fparser.scripts.fparser2:main
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - six
+  run:
+    - python
+    - six
+
+test:
+  imports:
+    - fparser
+    - fparser.common
+    - fparser.common.tests
+    - fparser.one
+    - fparser.one.tests
+    - fparser.scripts
+    - fparser.two
+  commands:
+    - fparser2 --help
+
+about:
+  home: https://github.com/stfc/fparser
+  license: BSD
```

SPDX identifiers are preferred:
```suggestion
  license: BSD-3-Clause
```
(I guess it's three clause but please check!)